pkg/network/istio/ports.go

```diff
@@ -42,3 +43,9 @@ func ReservedPorts() []string {
 		fmt.Sprint(EnvoyPrometheusTelemetryPort),
 	}
 }
+
+func NonProxiedPorts() []string {
+	return []string{
```

Comment:
Why not slice of integers?

Reply:
I wanted to keep it consistent with the ReservedPorts function